WeatherStation/v2/_init.yaml

```diff
@@ -0,0 +1,6 @@
+# A configuration that may be suitable for testing.
```

Comment:
I think this config needs to be change to something like this (assuming all config goes in the CSC schema, instead of having weather-station-type-specific config schema):
```
instances:
  - sal_index: 1
    type: lsst
    lsst_config:
      host: localhost
      port: 50000
      buffer_size: 2048
      timeout: 120.0
```
and if you ever do add support for other types of weather station then this file will need to be expanded to include default values for all those other types of weather station.

If, on the other hand, you prefer that your weather station class specifies its own config schema, then this becomes:
```
instances:
  - sal_index: 1
    type: lsst
    config:  # this and the next line are optional
        host: localhost
```
the lsst-specific schema can provide reasonable defaults for everything except host (and it can even use a default of 127.0.0.1 for the host, though that isn't really usable).

Reply:
Done